pkg/manager/internal.go

```diff
@@ -101,6 +108,16 @@ type controllerManager struct {
 	host string
 
 	webhookServer *webhook.Server
+
+	// leaseDurationSeconds is the duration that non-leader candidates will
+	// wait to force acquire leadership.
+	leaseDurationSeconds time.Duration
```

Comment:
drop the second suffix, and the `* time.Second` below. Durations have units already.